### PR TITLE
Extend Application Template with labels

### PR DESCRIPTION
**Pull request by kaloyanspiridonov**
**Description**

Changes proposed in this pull request:
- extend application template with labels
    - add new "app_template_id" column to labels table
    - add labels to graphql ApplicationTemplate and ApplicationTemplateInput
    - adopt in domain, model ...

**Pull Request status**

- [X] Implementation
- [X] Unit tests
- [X] Integration tests
- [X] `chart/compass/values.yaml` is updated 
- [X] Mocks are regenerated, using the automated script

### CODE REVIEW

File: components/director/internal/domain/apptemplate/converter.go [hunk 1]
```diff
@@ -95,12 +95,18 @@ func (c *converter) InputFromGraphQL(in graphql.ApplicationTemplateInput) (model
...
+		labels = in.Labels
+	}
+
 	return model.ApplicationTemplateInput{
 		Name:                 in.Name,
 		Description:          in.Description,
 		ApplicationInputJSON: appCreateInput,
 		Placeholders:         c.placeholdersFromGraphql(in.Placeholders),
 		AccessLevel:          model.ApplicationTemplateAccessLevel(in.AccessLevel),
+		Labels:               labels,
```
Comment (dzahariev) [inline components/director/internal/domain/apptemplate/converter.go:109]: What about UpdateInputFromGraphQL, if labels are updated they will not be applied without adoption there?
The same is for From/To entity functions too.

File: components/director/internal/domain/apptemplate/resolver.go [hunk 2]
```diff
@@ -211,6 +217,45 @@ func (r *Resolver) CreateApplicationTemplate(ctx context.Context, in graphql.App
...
+			resultLabels[label.Key] = label.Value
+		}
+	}
+
+	var gqlLabels graphql.Labels = resultLabels
+	return gqlLabels, nil
+}
+
 // RegisterApplicationFromTemplate missing godoc
 func (r *Resolver) RegisterApplicationFromTemplate(ctx context.Context, in graphql.ApplicationFromTemplateInput) (*graphql.Application, error) {
```
[the remark this answers is not in the document]
Comment (kaloyanspiridonov) [reply components/director/internal/domain/apptemplate/resolver.go:260]: Update of Application labels is not supported (check ApplicationUpdateInput). Do we need to support update of Application Template labels?

File: components/director/internal/domain/apptemplate/converter.go (hunk 1 above)
Comment (kaloyanspiridonov) [reply components/director/internal/domain/apptemplate/converter.go:109]: after offline discussion, we've decided to implement updating of labels in additional task

File: components/director/internal/domain/apptemplate/resolver.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (kaloyanspiridonov) [reply components/director/internal/domain/apptemplate/resolver.go:260]: after offline discussion, we've decided to implement updating of labels in additional task

**Review by desislavaa — DISMISSED** [review]

**Review by dzahariev — APPROVED** [review]

**Review by PetarTodorovv — DISMISSED** [review]

**Review by dzahariev — APPROVED** [review]

**Review by desislavaa — APPROVED** [review]

**Review by PetarTodorovv — APPROVED** [review]

**Pull request merged**